sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_receiver_async.py

```diff
@@ -69,11 +72,11 @@ async def _incoming_transfer(self, frame):
                 self._received_payload = bytearray()
             else:
                 message = decode_payload(frame[11])
-            delivery_state = await self._process_incoming_message(frame, message)
+            delivery_state = await self._process_incoming_message(self._first_frame, message)
```

Comment:
Do we need to reset `self._first_frame`? Or will it not matter?

Reply:
It shouldn't matter unless we do ^ as said above in previous comment. Every new message should have a new delivery_id